samples/bootloader/src/main.c

```diff
@@ -189,6 +189,23 @@ void main(void)
 	}
 #endif /* CONFIG_SOC_NRF9160 */
 
-	boot_from((u32_t *)s0_address_read());
+	const struct fw_firmware_info *s0_fw_info = fw_firmware_info_get(s0_address_read());
```

Comment:
Do we want to change these to `PM_` defines or should we still use the data stored in the provisioned section?

Reply:
To me it looks more "flexible" to use the data in the provisioned section.

The names of the PM partitions might not be directly map to what is expected here, so I say keep it like this.